### PR TITLE
[ZIPFLDR] Show icon on the taskbar

### PR DESCRIPTION
**Pull request by TAN-Gaming**
## Proposed changes
-  Show icon on the taskbar

VirtualBox_ReactOS_08_09_2022_09_43_14

### CODE REVIEW

**Review by katahiromz — APPROVED** [review]

File: dll/shellext/zipfldr/resource.h [hunk 1]
```diff
@@ -5,6 +5,9 @@
 #define IDB_HEADER                     200
 #define IDB_WATERMARK                  201
 
+/* Icons */
+#define IDI_ZIPFLDR 202
```
Comment (gonzoMD) [inline dll/shellext/zipfldr/resource.h:9]: Please use the same indent like the other numbers.

File: dll/shellext/zipfldr/zipfldr.rc [hunk 2]
```diff
@@ -6,7 +6,7 @@
 164 BITMAP "res/zipfldr.bmp"
 200 BITMAP "res/header.bmp"
 201 BITMAP "res/watermark.bmp"
-1 ICON "res/zipfldr.ico"
+IDI_ZIPFLDR ICON "res/zipfldr.ico"
```
Comment (gonzoMD) [inline dll/shellext/zipfldr/zipfldr.rc:9]: I know it's out of scope, but while you are already editing this file you could also exchange the other numbers with their identifiers. This is just bonus, not necessary for approvement.

**Review by gonzoMD — APPROVED** [review]
In general lgtm. See comments for minor stuff.

File: dll/shellext/zipfldr/resource.h [hunk 3]
```diff
@@ -1,34 +1,37 @@
...
-#define IDC_MESSAGE                 1206
+#define IDD_CONFIRM_FILE_REPLACE 1200
+#define IDYESALL                 1202
+#define IDC_EXCLAMATION_ICON     1205
+#define IDC_MESSAGE              1206
 
-#define IDD_PASSWORD                1300
-#define IDSKIP                      1301
+#define IDD_PASSWORD             1300
+#define IDSKIP                   1301
```
Comment (learn-more) [inline dll/shellext/zipfldr/resource.h:34]: Why is all this stuff edited?

**Review by learn-more — CHANGES_REQUESTED** [review]

File: dll/shellext/zipfldr/resource.h (hunk 3 above)
Comment (TAN-Gaming) [reply dll/shellext/zipfldr/resource.h]: Reverted.

File: dll/shellext/zipfldr/resource.h (hunk 1 above)
Comment (TAN-Gaming) [reply dll/shellext/zipfldr/resource.h]: Done.

File: dll/shellext/zipfldr/zipfldr.rc (hunk 2 above)
Comment (TAN-Gaming) [reply dll/shellext/zipfldr/zipfldr.rc:9]: Done.

File: dll/shellext/zipfldr/resource.h [hunk 4]
```diff
@@ -5,8 +5,11 @@
 #define IDB_HEADER                     200
 #define IDB_WATERMARK                  201
 
+/* Icons */
+#define IDI_ZIPFLDR                    202
```
Comment (HBelusca) [inline dll/shellext/zipfldr/resource.h:9]: Is it possible to use the original value of "1" ?

**Review by HBelusca — APPROVED** [review]
lgtm

File: dll/shellext/zipfldr/resource.h (hunk 4 above)
Comment (TAN-Gaming) [reply dll/shellext/zipfldr/resource.h]: Tried ID = 1, it works so i think it's OK.

**Review by learn-more — APPROVED** [review]

**Pull request merged**